lib/matplotlib/tests/test_axes.py

```diff
@@ -3437,14 +3437,17 @@ def test_errobar_nonefmt():
         assert np.all(errbar.get_color() == mcolors.to_rgba('C0'))
 
 
-def test_errorbar_fillstyle():
-    # Check that passing 'fillstyle' keyword will not result in errors
+def test_errorbar_fillstyle_drawstyle():
+    # Check that passing 'fillstyle' and 'drawstyle' keyword will not
+    # result in errors
     x = np.arange(5)
     y = np.arange(5)
 
     plotline, _, _ = plt.errorbar(x, y, xerr=1, yerr=1, ls='None',
-                                  marker='s', fillstyle='full')
+                                  marker='s', fillstyle='full',
+                                  drawstyle='steps-mid')
```

Comment:
Is it worth adding here keyword arguments for all the style arguments that you are `pop`ping above?

Reply:
OK, all options are added now.